src/main/java/net/minestom/server/item/metadata/FireworkMeta.java

```diff
@@ -17,11 +18,11 @@ public record FireworkMeta(TagReadable readable) implements ItemMetaView<Firewor
             .path("Fireworks").list().defaultValue(List.of());
     private static final Tag<Byte> FLIGHT_DURATION = Tag.Byte("Flight").path("Fireworks");
 
-    public List<FireworkEffect> getEffects() {
+    public @NotNull List<FireworkEffect> getEffects() {
         return getTag(EFFECTS);
     }
 
-    public byte getFlightDuration() {
+    public @Nullable Byte getFlightDuration() {
```

Comment:
Shouldn't this be defaulted to `0`? Should check vanilla.

Reply:
No, it is defaulted to null. The client will even display lore on the firework item if the flight duration is set